perf-measures/bundle-check/scripts/check-bundle-size.ts

```diff
@@ -0,0 +1,43 @@
+import * as path from 'path'
+import * as os from 'os'
+import * as fs from 'node:fs'
+import * as esbuild from 'esbuild'
+
+async function main() {
+  const tempDir = os.tmpdir()
+  const tempFilePath = path.join(tempDir, 'bundle.tmp.js')
+
+  try {
+    await esbuild.build({
+      entryPoints: ['dist/index.js'],
+      bundle: true,
+      minify: true,
+      format: 'esm' as esbuild.Format,
+      target: 'es2022',
+      outfile: tempFilePath,
+    })
```

Comment:
I have already confirmed that the results are the same as the CLI run, but is this build option appropriate?

Reply:
Yes. This is good.